websocket_server/websocket_server.py

```diff
@@ -290,7 +292,12 @@ def send_text(self, message, opcode=OPCODE_TEXT):
             raise Exception("Message is too big. Consider breaking it into chunks.")
             return
 
-        self.request.send(header + payload)
+	self.text_to_send.insert(0, header + payload)
```

Comment:
tabs mixed with spaces

Reply:
sorry for that. I edit the code directly on the GitHub webpage. Did not notice the problem of indentation. It's fixed now.